mlrun/model_monitoring/applications/base.py

```diff
@@ -220,8 +221,9 @@ def evaluate(
         :param with_repo:         Whether to clone the current repo to the build source.
         :param requirements:      List of Python requirements to be installed in the image.
```

Comment:
On the function docstring you sometimes mentions its Optional and sometime not although it's optional - please be consistent.
Since everything in this function seems optional, maybe (for clarity) have one note in the top of the docstring

Reply:
Done